From ca8b0903e890d57c3129c1bb60b8d02d6f848de5 Mon Sep 17 00:00:00 2001
From: lwan89x <liangx.wang@intel.com>
Date: Fri, 20 Jul 2012 09:55:43 +0800
Subject: [PORT FROM MAIN] Add libenc for UI optimizing in x86 platform.

BZ: 46856

Change-Id: I56c592b9584066606745baaebd30d65d461c5972
Orig-Change-Id: I6d23480ff253c350c75f396702209b0bdb51f839
Category: aosp improvement
Domain: PUPDR-Recovery
Origin: internal
Upstream-Candidate: no, need rework
Orig-Change-Id: Ib11faa5cf81f86e451b3ad4408f9ea13fdf7dc63
Signed-off-by: lwan89x <liangx.wang@intel.com>
---
 Android.mk |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/Android.mk b/Android.mk
index 075fa2c..5af5386 100644
--- a/Android.mk
+++ b/Android.mk
@@ -72,6 +72,11 @@ else
   LOCAL_STATIC_LIBRARIES += $(TARGET_RECOVERY_UI_LIB)
 endif
 
+#libpixelflinger_static for x86 is using encoder under hardware/intel/apache-harmony
+ifeq ($(TARGET_ARCH),x86)
+LOCAL_STATIC_LIBRARIES += libenc
+endif
+
 LOCAL_C_INCLUDES += system/extras/ext4_utils
 
 include $(BUILD_EXECUTABLE)
-- 
1.7.4.1

